src/hats_import/margin_cache/margin_cache.py

```diff
@@ -39,6 +39,11 @@ def generate_margin_cache(args, client):
             )
         resume_plan.wait_for_mapping(futures)
 
+    with resume_plan.print_progress(total=1, stage_name="Binning") as step_progress:
+        total_rows = resume_plan.get_mapping_total()
+        if not total_rows:
+            raise ValueError("Margin cache contains no rows. Increase margin size and re-run.")
```

Comment:
```suggestion
            raise ValueError("Margin cache contains no rows.  Increase margin size and re-run.")
```
Such a nitpick.  Sorry!  Just the way I was raised when writing with monospaced fonts for monospaced display.  🤦  Feel free to ignore, particularly if it contravenes existing message style in this project.

Reply:
Other messages and documentation in the project use a single space. This is now the recommendation of the Chicago Manual of Style, and several other style guides. Within the team, we defer to the Rubin DM developer style guide, which has some opinions, [at least for docstrings](https://developer.lsst.io/python/style.html#sentences-in-comments-should-not-be-separated-by-double-spaces), but also prefer single-space after sentence.